src/appshell/view/appmenumodel.cpp

```diff
@@ -457,7 +457,8 @@ MenuItem* AppMenuModel::makeDiagnosticsMenu()
             makeMenuItem("color-segment-shapes"),
             makeMenuItem("show-skylines"),
             makeMenuItem("show-system-bounding-rects"),
-            makeMenuItem("show-corrupted-measures")
+            makeMenuItem("show-corrupted-measures"),
+            makeMenuItem("check-sanity")
```

Comment:
This would now only be available in dev mode, while it would be useful in normal mode too. Let's make the Diagnostics > Engraving menu available in normal mode too; in normal mode, it should only contain the `show-corrupted-measures` and `check-sanity` actions.

Reply:
I don't get this

Reply:
```c++
if (globalConfiguration()->devModeEnabled()) {
   // show all those submenus, including the full version of the Engraving submenu
} else {
    // show only an Engraving submenu, that contains only `show-corrupted-measures` and `check-for-score-corruptions
}
```